feat(react-chart): add animation on transition (#2334)
DevExpress/devextreme-reactive: packages/dx-react-chart/docs/reference/animation.md

```js
import { Animation } from '@devexpress/dx-react-chart';
```

## User Reference

### Properties

Name | Type | Default | Description
-----|------|---------|------------

> easing? | ( t: number ) => number | | Easing functions for smooth animation.

Reviewer: ```suggestion
easing? | ( t: number ) => number | | An easing function for smooth animation.
```

> duration? | number | 1000 | Animation's duration.

Reviewer: ```suggestion
duration? | number | 1000 | Animation duration.
```

> duration | number | 1000 | Duration for the animation.

Reviewer 2: Are these properties required?

> duration? | number | 1000 | Duration for the animation.

Reviewer: ```suggestion
duration? | number | 1000 | Animation's duration.
```
